apps/rush-lib/src/logic/deploy/DeployManager.ts

```diff
@@ -133,85 +136,131 @@ export class DeployManager {
   private _collectFoldersRecursive(packageJsonFolderPath: string, deployState: IDeployState): void {
     const packageJsonRealFolderPath: string = FileSystem.getRealPath(packageJsonFolderPath);
 
-    if (!deployState.foldersToCopy.has(packageJsonRealFolderPath)) {
-      deployState.foldersToCopy.add(packageJsonRealFolderPath);
+    if (deployState.foldersToCopy.has(packageJsonRealFolderPath)) {
+      // we've already seen this folder
+      return;
+    }
```

Comment:
I de-dendeted this since the entire function was inside of this `if` block. I believe this change is functionally equivalent but if it isn't or you prefer to keep everything in this `if` block, I can revert this.

Reply:
I like it -- it's more readable